var/spack/repos/builtin/packages/improved-rdock/package.py

```diff
@@ -82,12 +92,12 @@ def test(self):
             "-s",
             "1",
         ]
-        self.run_test(str(mpiexe), options=opts)
-
-        opts = [join_path(self.test_suite.current_test_data_dir, "test.sh")]
-        self.run_test("bash", options=opts)
+        self.run_rdock_test(str(self.spec["mpi"].prefix.bin.mpirun), opts)
 
-        pythonexe = self.spec["python"].command.path
+    def test_pythonexe(self):
```

Comment:
It is not safe to assume this is the last test method that will be executed if it is pulled out even though it needs to be *after* running the `test.sh` script since that script produces the required output.

```
$ spack load improved-rock
$ python sdrmsd --help
Usage: sdrmsd [options] reference.sdf input.sdf

Options:
  -h, --help            show this help message and exit
  -f, --fit             Superpose molecules before RMSD calculation
  -t THRESHOLD, --threshold=THRESHOLD
                        Discard poses with RMSD < THRESHOLD with respect
                        previous poses which where not rejected based on same
                        principle. A Population SDField will be added to
                        output SD with the population number.
  -o FILE, --out=FILE   If declared, write an output SDF file with the input
                        molecules with a new sdfield <RMSD>. If molecule was
                        fitted, the fitted molecule coordinates will be saved.
Args:
        reference.sdf                SDF file with the reference molecule.
        input.sdf                SDF file with the molecules to be compared to reference.
```